mozilla/fx-private-relay
# MPP-2548 - Update Content in settings about label syncing

This PR fixes MPP-2548
 


# New feature description

Update Content in settings about label syncing 

<img width="974" alt="image" src="https://user-images.githubusercontent.com/3924990/210108894-2edc5f8d-85a9-4f27-a1dd-82592b6fd3ff.png">

<img width="474" alt="image" src="https://user-images.githubusercontent.com/3924990/210109720-64d8c911-0ddb-422f-9811-ba3229b5b5f1.png">
 
# How to test

Visit the settings page while logged in. Toggle privacy setting to see warnings. Text should match screenshot. 

Visit the FAQ page, check content under question "What data does ⁨Firefox Relay⁩ collect?"

# Checklist (Definition of Done)
